src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
+        Lottery[] someBalls = new Lottery[NUMBER_OF_BALLS];
+        for (int i = 0; i < someBalls.length; i++) {
+            someBalls[i] = new Lottery();
```

Comment:
[Don’t create several instances of the same class if you can use only one instance for your purposes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes) You need to create 3 balls not 3 instances of Lottery

Reply:
Fixed it, thank you